src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/session/GetSessionCapacityTest.kt

```diff
@@ -0,0 +1,105 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.integration.session
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.web.reactive.server.WebTestClient.BodyContentSpec
+import org.springframework.test.web.reactive.server.WebTestClient.ResponseSpec
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.SessionCapacityDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.controller.GET_SESSION_CAPACITY
+import uk.gov.justice.digital.hmpps.visitscheduler.integration.IntegrationTestBase
+import java.time.LocalDate
+import java.time.LocalTime
+
+@DisplayName("Get /visit-sessions")
+class GetSessionCapacityTest(@Autowired private val objectMapper: ObjectMapper) : IntegrationTestBase() {
+
+  private val requiredRole = listOf("ROLE_VISIT_SCHEDULER")
+
+  @Test
+  fun `get session capacity`() {
```

Comment:
We might need to add a test for bi-weekly session templates.

Reply:
true